docs/table/index.rst

```diff
@@ -7,24 +7,22 @@ Data Tables (`astropy.table`)
 Introduction
 ============
 
-`astropy.table` provides functionality for storing and manipulating
-heterogeneous tables of data in a way that is familiar to ``numpy`` users. A few
-notable capabilities of this package are:
-
-* Initialize a table from a wide variety of input data structures and types.
-* Modify a table by adding or removing columns, changing column names,
-  or adding new rows of data.
-* Handle tables containing missing values.
-* Include table and column metadata as flexible data structures.
-* Specify a description, units, and output formatting for columns.
-* Interactively scroll through long tables similar to using ``more``.
-* Create a new table by selecting rows or columns from a table.
+`astropy.table` provides a flexible and easy-to-use set of tools for working with
+tabular data using an interface based on `numpy`. In addition to basic table creation,
+access, and modification operations, key features include:
+
+* Support columns of astropy :ref:`time <astropy-time>`, :ref:`coordinates <astropy-coordinates>`, and :ref:`quantities <quantity>`.
+* Support multidimensional and :ref:`structured array columns <structured-array-as-a-column>`.
+* Maintain the units, description, and format of columns.
+* Provide flexible metadata structures for the table and individual columns.
 * Perform :ref:`table_operations` like database joins, concatenation, and binning.
 * Maintain a table index for fast retrieval of table items or ranges.
-* Manipulate multidimensional and :ref:`structured array columns <structured-array-as-a-column>`.
-* Handle non-native (mixin) column types within table.
-* Methods for :ref:`read_write_tables` to files.
-* Hooks for :ref:`subclassing_table` and its component classes.
+* Support a general :ref:`mixin protocol <mixin_protocol>` for flexible data containers in tables, including non-astropy objects.
+* :ref:`Read and write <read_write_tables>` to files via the :ref:`Unified File Read/Write Interface <table_io>`.
+* Convert to and from `pandas.DataFrame`.
+
+The :ref:`astropy-table-and-dataframes` page provides the rationale for maintaining
```

Comment:
I'd move this as part of the item just above -- maybe 
```
* Convert to and from `pandas.DataFrame` (see :ref:`pandas`
 as well as :ref:`astropy-table-and-dataframes` for the rationale
 for a dedicated `astropy.table` package instead of relying on `pandas`).
```
(not sure I got the references quite right - want to have a direct link to `pandas.rst` as well...)

p.s. Fine to ignore!

Reply:
Looking at the rendered HTML again, I saw that every list item was succinct and fit on one line, with the exception of the one about mixins. So instead of making the pandas line longer, I shortened the mixin line and now it looks very clean.

This of course leaves the part about astropy-table-and-dataframes as a distinct coda to the introduction, but this comes up so often that it seems OK.